src/test/java/seedu/address/logic/commands/FindCommandTest.java

```diff
@@ -26,6 +28,8 @@ public class FindCommandTest {
     private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
     private Model expectedModel = new ModelManager(getTypicalAddressBook(), new UserPrefs());
 
+    private Model modelWithRental = new ModelManager(getTypicalAddressBookWithRental(), new UserPrefs());
+    private Model expectedModelWithRental = new ModelManager(getTypicalAddressBookWithRental(), new UserPrefs());
     @Test
     public void equals() {
         NameContainsKeywordsPredicate namePredicate1 =
```

Comment:
May be good to change the name to something like firstNamePredicate to match with findFirstCommand used later.

Reply:
I will keep the current name for now! I will keep in view for future changes! 

Thanks for pointing that out!